english/bored/bored.md

```diff
@@ -9,3 +9,5 @@
 "Or should I go on an [ADVENTURE!](../eat-rainbows/rainbows.md)?"
 
 "Or maybe [escape to the wilderness](../escape-to-the-wilderness/wilderness.md)?"
+
+["Maybe I should go to a jazz club?"](../jazz-guitar.jazz.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.